app/services/taxonomy/publish_taxon.rb

```diff
@@ -25,6 +25,14 @@ def publish
       raise(InvalidTaxonError, I18n.t('errors.invalid_taxon'))
     end
 
+    def unpublish(type:, alternative_path: nil)
```

Comment:
What do you think of using a different class for this? This is called `PublishTaxon`, and having an `unpublish` method could be a bit confusing.